### PR TITLE
[zh] Resync kubelet reference

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Review by tanjunchen — COMMENTED** [review]
/approve

File: content/zh/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -691,10 +691,11 @@ If &gt; `0`, limit event creations per second to this value. If `0`, unlimited.
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-A set of eviction thresholds (e.g. `memory.available<1Gi`) that if met would trigger a pod eviction. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's `--config` flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)
+A set of eviction thresholds (e.g. `memory.available<1Gi`) that if met would trigger a pod eviction. On a Linux node, the default value also includes `nodefs.inodesFree<5%`. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's `--config` flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)
 -->
 触发 Pod 驱逐操作的一组硬性门限（例如：<code>memory.available&lt;1Gi</code>
-（内存可用值小于 1 G））设置。
+（内存可用值小于 1 G））设置。在 Linux 节点上，默认值还包括
+<code>nodefs.inodesFree<5%`。
```
Comment (howieyuen) [inline content/zh/docs/reference/command-line-tools-reference/kubelet.md:698]: need \</code\>

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/command-line-tools-reference/kubelet.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/reference/command-line-tools-reference/kubelet.md:698]: :+1:

Comment (tengqm) [reply content/zh/docs/reference/command-line-tools-reference/kubelet.md:698]: good spot.

**Pull request merged**